components/LanguageSwitcher.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="host">
+    <GlobeIcon width="2.2rem" height="2.2rem" class="globe" @click="toggle" />
+    <div v-if="active" class="languages">
+      <nuxt-link
+        v-for="language in languages"
+        :key="language.id"
+        :to="switchLocalePath(language.id)"
+        class="languageItem"
+      >
+        {{ language.name }}
+      </nuxt-link>
+    </div>
+  </div>
+</template>
+
+<script>
+import GlobeIcon from '@/assets/icons/globe.svg'
+
+export default {
+  name: 'LanguageSwitcher',
+  components: { GlobeIcon },
+  data () {
+    return {
+      active: false,
+      languages: [
+        { id: 'en', name: 'English' },
+        { id: 'de', name: 'Deutsch' }
+      ]
+    }
+  },
+  methods: {
+    toggle () {
+      this.active = !this.active
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+.host {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.languages {
+  top: 70px;
+  position: fixed;
+  margin-top: 10px;
+  display: flex;
+  flex-direction: column;
+
+  background-color: rgba(255, 255, 255, 0.25);
```

Comment:
background color should have a higher alpha value. With 0.25 the text of the language switcher is not easy readable if there was other text (for example the typed slogans) in the background